### DISCUSSION TITLE
How to use SCSS in Custom Extension?

### DISCUSSION THREAD

**Discussion by haidv1992** [directus/directus#7916] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

```
haidv@haidvs-iMac dashboard % npx directus-extension build
⠋ Building Directus extension...(node:54486) UnhandledPromiseRejectionWarning: Error: Cannot find module 'sass'
Require stack:
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/rollup-plugin-vue/dist/sfc.js
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/rollup-plugin-vue/dist/index.js
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/@directus/extensions-sdk/dist/cjs/cli/commands/build.js
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/@directus/extensions-sdk/dist/cjs/cli/run.js
- /usr/local/var/www/directus/directus-project/src/dashboard/node_modules/@directus/extensions-sdk/cli.js
```

https://user-images.githubusercontent.com/86423592/132435236-55a91db2-dbbd-434b-85bb-ab69b432ac9f.mov

### To Reproduce

create module extension 
style with lang="scss"
npx directus-extension build

### What version of Directus are you using?

v9.0.0-rc.92

### What version of Node.js are you using?

v14.17.1

### What database are you using?

Mysql

### What browser are you using?

chorme

### What operating system are you using?

macOS

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
@nickrum Any insight?

**Reply by alesvaupotic** [reply]
`npm i sass` won't solve it? It works here: 

![image](https://user-images.githubusercontent.com/1660347/132526369-680377a5-30fe-42c9-b8eb-b82662ce3dfd.png)

**Reply by AustinPhillipTaylor** [reply]
I believe this is correct, the `Cannot find module 'sass'` is the error Rollup always throws when it can't find a sass installation.

**Reply by nickrum** [reply]
We are using `rollup-plugin-styles` internally to bundle stylesheets. They support `sass` and `node-sass`, so this should in theory work.

**Reply by rijkvanzanten** [reply]
@nickluger should work ootb, or should work after manually adding `sass` as a dependency? (fwiw `node-sass` is deprecated in favor of `sass`

**Reply by nickrum** [reply]
Should work ootb with only `node-sass` installed. But, as it is deprecated, I would recommend to use `sass` instead.

**Reply by rijkvanzanten** [reply]
I see, so in this case, the OP tried using sass without having the sass dependency installed altogether, which fails as expected 👍🏻

**Comment by nickrum** [comment]
The vue compiler specifically needs `sass` (instead of `node-sass`) to be able to compile scss.
